WordPress/metadata/release_notes.txt

```diff
@@ -1,2 +1,3 @@
-We added color/background customization for text blocks.
-It wasn't possible for unsupported blocks to be edited with accounts created before December 2018. We fixed that! Along the way, we also updated loading/failed screens for the unsupported block editor.
\ No newline at end of file
+* [***] Block editor: New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
+* [*] Added webp image format support [https://github.com/wordpress-mobile/WordPress-Android/pull/15068]
+
```

Comment:
Those might be incomplete, as Gio noted, and people are adding some forgotten notes by commenting on the Editorial post (p2JRYi-5r4-p2) before we get the final copy.